lib/Agrammon/OutputFormatter/Excel.pm6

```diff
@@ -97,22 +103,22 @@ sub input-output-as-excel(
         $row++;
 
         # formatted data
-        my $instance = %rec<instance>;
-        my $module = %rec<gui>;
-        if $module ne $last-module {
-            $input-sheet-formatted.set($row-formatted, $col+0, $module, :bold);
-            $row-formatted++;
-            $last-module = $module;
-        }
-        if $instance and $instance ne $last-instance {
-            $input-sheet-formatted.set($row-formatted, $col+1, $instance, :bold);
-            $row-formatted++;
-            $last-instance = $instance;
-        }
-        $input-sheet-formatted.set($row-formatted, $col+1, %rec<input>);
-        $input-sheet-formatted.set($row-formatted, $col+2, (%rec<value-translated> // '???'), :number-format('#,#'), :horizontal-align(RightAlign));
-        $input-sheet-formatted.set($row-formatted, $col+3, %rec<unit>);
-        $row-formatted++;
+#        my $instance = %rec<instance>;
+#        my $module = %rec<gui>;
+#        if $module ne $last-module {
+#            $input-sheet-formatted.set($row-formatted, $col+0, $module, :bold);
+#            $row-formatted++;
+#            $last-module = $module;
+#        }
+#        if $instance and $instance ne $last-instance {
+#            $input-sheet-formatted.set($row-formatted, $col+1, $instance, :bold);
+#            $row-formatted++;
+#            $last-instance = $instance;
+#        }
+#        $input-sheet-formatted.set($row-formatted, $col+1, %rec<input>);
+#        $input-sheet-formatted.set($row-formatted, $col+2, (%rec<value-translated> // '???'), :number-format('#,#'), :horizontal-align(RightAlign));
+#        $input-sheet-formatted.set($row-formatted, $col+3, %rec<unit>);
+#        $row-formatted++;
```

Comment:
Is this really meant to be in the PR?

Reply:
At the moment yes, will go back in once the testing is over (not needed there and later the performance impact doesn't matter so much). Added a comment about it.